src/components/views/rooms/EventTile.tsx

```diff
@@ -1092,7 +1092,7 @@ export default class EventTile extends React.Component<IProps, IState> {
                         onClick={this.onRoomClicked}
                     >
                         <RoomAvatar room={room} width={28} height={28} />
-                        { room ? room.name : '' }
+                        { room?.name || '' }
```

Comment:
you also shouldn't need the `||` anymore, because undefined/null gets rendered as an empty string.

Reply:
:facepalm: